plugins/hydra_optuna_sweeper/example/conf/config.yaml

```
@@ -4,14 +4,14 @@ defaults:
 
 hydra:
   sweeper:
+    sampler:
+      seed: 123
     optuna_config:
       direction: minimize
       study_name: sphere
       storage: null
       n_trials: 20
       n_jobs: 1
```

Okay, I think we can completely kill optuna_config here and go for this:
```suggestion
hydra:
  sweeper:
    sampler:
      seed: 123
    direction: minimize
    study_name: sphere
    storage: null
    n_trials: 20
    n_jobs: 1
```

---

will do